lib/modules/manager/regex/utils.ts

```diff
@@ -97,3 +98,16 @@ export function mergeExtractionTemplate(
     replaceString: addition.replaceString ?? base.replaceString,
   };
 }
+
+export function isValidDependency({
+  depName,
+  currentValue,
+  currentDigest,
+}: PackageDependency): boolean {
+  // check if all the fields are set
+  return (
+    (is.nonEmptyStringAndNotWhitespace(depName) ||
+      is.nonEmptyStringAndNotWhitespace(currentDigest)) &&
+    is.nonEmptyStringAndNotWhitespace(currentValue)
```

Comment:
```suggestion
    is.nonEmptyStringAndNotWhitespace(depName) &&
    (is.nonEmptyStringAndNotWhitespace(currentDigest) ||
    is.nonEmptyStringAndNotWhitespace(currentValue))
```
Seems to more logical to me

Reply:
:man_facepalming: